util/flocks/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "flocks"
+version = { workspace = true }
+edition = { workspace = true }
+license = { workspace = true }
+authors = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+publish = { workspace = true }
+rust-version = { workspace = true }
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+serde = { workspace = true, features = ["derive"] }
+anyhow = { workspace = true }
+tokio = { workspace = true }
+futures = { workspace = true }
+rustix = { workspace = true, features = ["fs"]}
```

Comment:
We're walking into Unix-specific territory here. For Windows, you'd have to use a different Win32 API. Hopefully this is intended.

Reply:
I am fine with not running blockchain on windows :)